redis/src/main/scala/akka/streams/alpakka/redis/model.scala

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright (C) 2016-2018 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.streams.alpakka.redis
+
+import java.util.Optional
+import scala.collection.immutable.Seq
+import scala.util.{Failure, Success, Try}
+import scala.collection.JavaConverters._
+
+final class RedisFieldValue[K, V] private (val field: K, val value: V) {
```

Comment:
Yes. This because of javadsl.  In my previous contribution with Couchbase intrgration enru asked my to use this structures instead of case class. In general every object that can be created by user using javadsl can't be case class.